### PR TITLE
vendor: update dependencies

### PR DESCRIPTION
**Pull request by fjl**
This is includes a fix for https://github.com/syndtr/goleveldb/issues/161. I'm not sure whether we're affected by this, but it's worth a try.

### CODE REVIEW

**Comment by fjl** [comment]
Looks like the azure sdk doesn't support go 1.5 anymore.

File: cmd/utils/customflags.go [hunk 1]
```diff
@@ -46,7 +44,6 @@ func (self *DirectoryString) Set(value string) error {
 // Custom cli.Flag type which expand the received string to an absolute path.
 // e.g. ~/.ethereum -> /home/username/.ethereum
 type DirectoryFlag struct {
-	cli.GenericFlag
```
Comment (karalabe) [inline cmd/utils/customflags.go]: This was actually fixed in my config pr that's pending for a couple months now O:)

Comment (fjl) [inline cmd/utils/customflags.go]: It's been pending since Dec 11, so exactly one month.

Comment (karalabe) [inline cmd/utils/customflags.go]: If pending since Dec 11, that's one month + 1 day. According to your logic, it's already a "couple" ;)

Comment (fjl) [inline cmd/utils/customflags.go]: What's one month?

Comment (karalabe) [inline cmd/utils/customflags.go]: If it was pending on Dec 11, and today it's Jan 11, then if Dec 11 was the first day of pending, today is the 32nd. That's more than a month by any measure :D

Comment (fjl) [inline cmd/utils/customflags.go]: OK, you're right.

Comment (karalabe) [inline cmd/utils/customflags.go]: Anyway, at least this issue is fixed now, will rebase my pr on top.

**Comment by karalabe** [comment]
Although a memo, I want us to run this codebase a bit before releasing since a depsupdate might have unexpected side effects.

**Pull request merged**